### ISSUE TITLE
I did read the Notes - but could not find a roadmap

### ISSUE THREAD

**Issue by pmario** [cozodb/cozo#45]
Hi, Very interesting project.  

I did have a look at: https://docs.cozodb.org/en/latest/release.html .. But I could not find a roadmap to V1. May be I've missed this info in the tutorial docs ... 

Thanks for sharing this project!

**Comment by pmario** [comment]
Forget the question. I found it :) https://docs.cozodb.org/en/latest/releases/v0.5.html#cozo-0-5-the-versatile-embeddable-graph-database-with-datalog-is-half-way-1-0

**Comment by zh217** [comment]
Glad that you found it! I will move the note to somewhere more prominent.

**Comment by pmario** [comment]
Thx for the fast response. I do have an other question about Cozo Script

At [Rules and relations](https://docs.cozodb.org/en/latest/tutorial.html#Rules-and-relations) you write

>Previous examples all start with ?[] <-. The notation denotes a rule named ?. It is a constant rule since it uses <-. The part after ? and before <- is the head of the rule, and the part after <- is the body.

Is there an overview about the differen rules. eg: What is the syntax for the rule. Or what is the syntax for the body and so on. 

I had quite a hard time to "map the queries and the resulting lists in my brain". Especially once the more interesting stuff with [graphs](https://docs.cozodb.org/en/latest/tutorial.html#Graphs) started. ... 

I got really lost at: https://docs.cozodb.org/en/latest/tutorial.html#Negation ... I had to rewrite the querys quite a bit to match my thinking. Especially "loved_by_b_e" and `b_e` caused a lot of problems until I understood that b->bob and e->eve. ... 

Is there a [railrod like](https://en.wikipedia.org/wiki/Syntax_diagram) syntax description somewhere?

**Comment by zh217** [comment]
Yes there is a chapter in the manual that deals with the syntax: https://docs.cozodb.org/en/latest/queries.html .

There is currently no formal syntax description. However, since the parser part is written in [pest](https://pest.rs/), you can read the [source code](https://github.com/cozodb/cozo/blob/main/cozo-core/src/cozoscript.pest), which is quite readable.
